packages/amplify-graphql-model-transformer/src/__tests__/__snapshots__/model-transformer-override.test.ts.snap

```diff
@@ -1472,12 +1473,13 @@ Object {
                 "dynamodb:PutItem",
                 "dynamodb:UpdateItem",
                 "dynamodb:DeleteItem",
+                "dynamodb:DescribeTable",
               ],
               "Effect": "Allow",
               "Resource": Array [
                 Object {
                   "Fn::GetAtt": Array [
-                    "CommentTable",
+                    "CommentTableE9F84112",
                     "Arn",
```

Comment:
Is the new construct creating new logicalIds for ddbTable?
That can create problem

Reply:
I logged a follow up to investigate this when we get to a stage when we can run e2e workflows (tests and manual). https://app.asana.com/0/1202389680978480/1203117329609220/f .
The name isn't pretty but might be ok given that it's consistent change across the whole file.